src/request.ts

```diff
@@ -132,17 +132,18 @@ export class OData {
    * internal csrf token
    */
   private csrfToken: string = "";
+  private sessionCookies : string = ""
   /**
    * dont direct use this object
    */
   private commonHeader: { [headerName: string]: string } = {
     "Accept": "application/json",
-    "Accept-Language": "zh",
+    "Accept-Language": "en-US",
     "Content-Type": "application/json"
   };
   private fetchProxy = odataDefaultFetchProxy;
   private processCsrfToken = true;
-  private forSAP = false;
+  private forSAP = true;
```

Comment:
Please do NOT change this default property value, in addition, this value could be set at the `OData` class constructor

Reply:
```suggestion
  private forSAP = false;
```